modules/2_nbde/files/powervs-setup.yml

```diff
@@ -82,6 +83,7 @@
           # Setup RHEL9 subscription
           os_ver=$(cat /etc/os-release | egrep "^VERSION_ID=" | awk -F'"' '{print $2}')
           if [[ $os_ver != "9"* ]]; then
+            sudo yum update -y
```

Comment:
What's the reason for the `yum update -y` here? 

sudo yum update -y at L86 is duplicate of https://github.com/IBM/powervs-tang-server-automation/blob/a941cae3fd8b8dc61176a73106dd9582f197266d/modules/2_nbde/files/powervs-setup.yml#L96

Reply:
Removed the line because its duplicate.